lib/utils.js

```diff
@@ -245,10 +243,12 @@ function setup(pluginInfo, options) {
       'ember-cli-htmlbars:inline-precompile',
     ];
   } else {
-    plugin = [
+    return [
       require.resolve('babel-plugin-ember-template-compilation'),
       {
-        precompile,
+        // As above, we present the AST transforms in reverse order
+        transforms: [...pluginInfo.plugins].reverse(),
```

Comment:
This new API uses forward order so the `reverse()` should not be needed.

Reply:
I'll double check, but without the `reverse` I believe I was seeing inverted execution order compared to what's on `master`

Reply:
Ok, well I trust your findings over my memory as long as you're checking.

Reply:
I think you're right, I think the plugin runs them in the order stated, but it's ember-cli-htmlbars that has always reversed them before handing them onward.

Reply:
Confirmed: if I register [the test plugin](https://github.com/ember-cli/ember-cli-htmlbars/blob/master/tests/dummy/lib/module-name-inliner/index.js#L12) twice and log from the `PathExpression` visitor, keeping the `reverse()` call preserves the same output ordering I see on `master`.